aml_vision/image-classification/README.md

```diff
@@ -0,0 +1,81 @@
+# AzureML Foundational Vision Models Fine-Tuning Examples
+
+This codebase shows how to use ACPT (Azure Container for PyTorch) along with accelerators such as ONNX Runtime Training (through Hugging Face Optimum) and DeepSpeed to fine-tune foundational, computer-vision models for an image classification task. Below are benchmarks on the DeepFashion dataset using our various accelerators:
+
+| Model                                                                                         | Max Batch Size | Accelerators | train_runtime (sec)           | train_samples_per_second      |
+|-----------------------------------------------------------------------------------------------|----------------|--------------|-------------------------------|-------------------------------|
+| [google/vit-base](https://huggingface.co/google/vit-base-patch16-224)                         | 117            | -            | 1306.508                      | 1065.129                      |
```

Comment:
Instead of "-" under accelerators use something like "default" and explain what default config is either above or below this table.

Reply:
@askhade I think adding "default" clogs up the table; however, I will add a note on what the difference between the no accelerator and ORT+DS runs are.